packages/functional-tests/tests/syncV3/signIn.spec.ts

```diff
@@ -151,22 +199,30 @@ test.describe('severity-2 #smoke', () => {
       target,
       syncBrowserPages,
     }) => {
-      const { page, login, connectAnotherDevice, settings, totp } =
-        syncBrowserPages;
+      const {
+        page,
+        login,
+        connectAnotherDevice,
+        settings,
+        totp,
+        signinTotpCode,
+      } = syncBrowserPages;
 
       await settings.goto();
       await settings.totp.clickAdd();
       const { secret } = await totp.fillOutTwoStepAuthenticationForm();
       credentials.secret = secret;
       await settings.signOut();
 
-      // Sync sign in
       await page.goto(
         `${target.contentServerUrl}?context=fx_desktop_v3&service=sync`,
         { waitUntil: 'load' }
       );
+
       await login.login(credentials.email, credentials.password);
-      await login.setTotp(credentials.secret);
+
+      const code = await getCode(credentials.secret);
```

Comment:
Noticing this imports the method from `fxa-settings`. Just double checking if we're ok with that 👍